ccl/changefeedccl: TestChangefeedOnlyInitialScan failed

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7555874?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7555874?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3703856319
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6571: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3703856319
--- FAIL: TestChangefeedOnlyInitialScan (91.76s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6560: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3141/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6560
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (44.90s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6560: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3141/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6560
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (42.70s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (91.55s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-21535

Epic CRDB-11732

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7574525?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7574525?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan4171277769
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6571: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan4171277769
--- FAIL: TestChangefeedOnlyInitialScan (89.96s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (89.83s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6553: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3041/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.54s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6560: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3041/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6560
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (39.94s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7593733?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7593733?buildTab=artifacts#/) on master @ [8357abb668a5adaff781343b394b162fb1b66c6e](https://github.com/cockroachdb/cockroach/commits/8357abb668a5adaff781343b394b162fb1b66c6e):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan156229157
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6571: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan156229157
--- FAIL: TestChangefeedOnlyInitialScan (52.25s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (52.00s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6527: error executing 'CREATE TABLE foo (a INT PRIMARY KEY)': pq: relation "d.public.foo" already exists
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (0.09s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6553: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3041/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (47.20s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7672932?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7672932?buildTab=artifacts#/) on master @ [39250f643837156c8fb75b0ced07ef8a99c14185](https://github.com/cockroachdb/cockroach/commits/39250f643837156c8fb75b0ced07ef8a99c14185):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1455684750
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6571: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1455684750
--- FAIL: TestChangefeedOnlyInitialScan (93.95s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (93.84s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6553: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2845/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.66s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6560: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2845/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6560
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (40.93s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92211 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7743040?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7743040?buildTab=artifacts#/) on master @ [2b595c5214aead3e7d4f47f9a1dedce57dba6e7f](https://github.com/cockroachdb/cockroach/commits/2b595c5214aead3e7d4f47f9a1dedce57dba6e7f):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3275942332
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6571: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3275942332
--- FAIL: TestChangefeedOnlyInitialScan (90.26s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (90.14s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6553: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6553
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.70s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6560: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3254/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6560
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (37.82s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92211 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7816518?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7816518?buildTab=artifacts#/) on master @ [ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410](https://github.com/cockroachdb/cockroach/commits/ca079ac1fc8a8f7a5360f5b1f4fa210c14dc8410):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3778198357
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6574: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3778198357
--- FAIL: TestChangefeedOnlyInitialScan (54.95s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3824/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (47.24s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (54.80s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6524: error executing 'CREATE TABLE foo (a INT PRIMARY KEY)': pq: relation "d.public.foo" already exists
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (0.10s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7852965?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7852965?buildTab=artifacts#/) on master @ [146556e19f5e4fdc8c3e6a623b280cc33aee4d18](https://github.com/cockroachdb/cockroach/commits/146556e19f5e4fdc8c3e6a623b280cc33aee4d18):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan81367251
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6574: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan81367251
--- FAIL: TestChangefeedOnlyInitialScan (98.09s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (97.90s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4122/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.83s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6563: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4122/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6563
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (44.53s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910089?buildTab=artifacts#/) on master @ [f2b00e8039af6ea8887ec124dad8daf19da6fbf1](https://github.com/cockroachdb/cockroach/commits/f2b00e8039af6ea8887ec124dad8daf19da6fbf1):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1468899888
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6574: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1468899888
--- FAIL: TestChangefeedOnlyInitialScan (96.58s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4127/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.15s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6563: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4127/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6563
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (43.41s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (96.40s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc deprecated-branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7916831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7916831?buildTab=artifacts#/) on master @ [c050c9b4b57ecf2ceb5d449c31c617fe12c920e0](https://github.com/cockroachdb/cockroach/commits/c050c9b4b57ecf2ceb5d449c31c617fe12c920e0):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2158467047
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6574: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2158467047
--- FAIL: TestChangefeedOnlyInitialScan (101.50s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (101.32s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3658/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (47.17s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6563: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3658/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6563
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (46.45s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc deprecated-branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924024?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7924024?buildTab=artifacts#/) on master @ [942a4d468e9c8ad0ef45a7be33f0a326dfb19fef](https://github.com/cockroachdb/cockroach/commits/942a4d468e9c8ad0ef45a7be33f0a326dfb19fef):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1152798471
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6574: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1152798471
--- FAIL: TestChangefeedOnlyInitialScan (95.67s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (95.54s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4127/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (48.42s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6563: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4127/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6563
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (41.12s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc deprecated-branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7953697?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7953697?buildTab=artifacts#/) on master @ [052acc88ad9d7296ce6b8b441627fb469cc74d95](https://github.com/cockroachdb/cockroach/commits/052acc88ad9d7296ce6b8b441627fb469cc74d95):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2055853848
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6567: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2055853848
--- FAIL: TestChangefeedOnlyInitialScan (69.11s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:754: making server as system tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (69.10s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4130/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (35.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8000746?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8000746?buildTab=artifacts#/) on master @ [7c8e7012b73a598f3aa1da9bbf422e833706f5d3](https://github.com/cockroachdb/cockroach/commits/7c8e7012b73a598f3aa1da9bbf422e833706f5d3):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1944977634
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6567: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1944977634
--- FAIL: TestChangefeedOnlyInitialScan (97.42s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (43.74s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (97.40s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6549: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.62s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8007514?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8007514?buildTab=artifacts#/) on master @ [93ed65565357538c9048ff45c878a493f2ed9b45](https://github.com/cockroachdb/cockroach/commits/93ed65565357538c9048ff45c878a493f2ed9b45):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1902329465
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6567: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan1902329465
--- FAIL: TestChangefeedOnlyInitialScan (98.65s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (98.63s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6549: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.88s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (44.27s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8031581?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8031581?buildTab=artifacts#/) on master @ [61f7c816574848f804f1adf7f42f7315bfcaaeba](https://github.com/cockroachdb/cockroach/commits/61f7c816574848f804f1adf7f42f7315bfcaaeba):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2726376721
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6567: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2726376721
--- FAIL: TestChangefeedOnlyInitialScan (90.54s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6556: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6556
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (38.67s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (90.53s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6549: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4125/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6549
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (46.29s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8101213?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8101213?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2754001539
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6568: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan2754001539
--- FAIL: TestChangefeedOnlyInitialScan (69.57s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6557: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4134/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6557
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (34.06s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:754: making server as system tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (69.56s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8117809?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8117809?buildTab=artifacts#/) on master @ [6723e00a46aaa3ea575093bd82a02b7d6f6b131b](https://github.com/cockroachdb/cockroach/commits/6723e00a46aaa3ea575093bd82a02b7d6f6b131b):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan4048660711
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6568: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan4048660711
--- FAIL: TestChangefeedOnlyInitialScan (99.43s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (99.41s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6550: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4024/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6550
        	Error:      	Received unexpected error:
        	            	still waiting for job status; current running
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:347
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:69
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsSoonError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:48
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.(*jobFeed).WaitForStatus
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/testfeed_test.go:338
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl.TestChangefeedOnlyInitialScan.func1.1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6550
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1446
        	            	  | runtime.goexit
        	            	  | 	GOROOT/src/runtime/asm_amd64.s:1594
        	            	Wraps: (2) still waiting for job status; current running
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (48.56s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6557: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4024/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6557
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (43.99s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedOnlyInitialScan [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8153934?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8153934?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):


```
=== RUN   TestChangefeedOnlyInitialScan
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3378052305
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedOnlyInitialScan
    changefeed_test.go:6568: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedOnlyInitialScan3378052305
--- FAIL: TestChangefeedOnlyInitialScan (81.66s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise
    helpers_test.go:751: making server as secondary tenant
    helpers_test.go:837: making enterprise feed factory
    helpers_test.go:851: pgURL enterprise root
    --- FAIL: TestChangefeedOnlyInitialScan/enterprise (81.65s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
    changefeed_test.go:6557: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3294/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6557
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_backfill_only (35.21s)
=== RUN   TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
    changefeed_test.go:6557: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/3294/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/changefeedccl/changefeedccl_test_/changefeedccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/changefeedccl/changefeed_test.go:6557
        	Error:      	Not equal: 
        	            	expected: 5000
        	            	actual  : 0
        	Test:       	TestChangefeedOnlyInitialScan/enterprise/initial_scan_only
        --- FAIL: TestChangefeedOnlyInitialScan/enterprise/initial_scan_only (41.85s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #92887 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2.0]
- #92811 ccl/changefeedccl: TestChangefeedOnlyInitialScan failed [C-test-failure O-robot T-cdc branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedOnlyInitialScan.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/cdc.go
+++ b/pkg/cmd/roachtest/tests/cdc.go
@@ -504,9 +504,7 @@ func newCDCTester(ctx context.Context, t test.Test, c cluster.Cluster) cdcTester
 		t.L().Printf("failed to set cluster setting: %s", err)
 	}
 
-	if !t.SkipInit() {
-		tester.startGrafana()
-	}
+	tester.startGrafana()
 	return tester
 }
 
@@ -520,15 +518,19 @@ func (ct *cdcTester) startGrafana() {
 
 	ct.promCfg = cfg
 
-	err := ct.cluster.StartGrafana(ct.ctx, ct.t.L(), cfg)
-	if err != nil {
-		ct.t.Errorf("error starting prometheus/grafana: %s", err)
+	if !ct.t.SkipInit() {
+		err := ct.cluster.StartGrafana(ct.ctx, ct.t.L(), cfg)
+		if err != nil {
+			ct.t.Errorf("error starting prometheus/grafana: %s", err)
+		}
+		nodeURLs, err := ct.cluster.ExternalIP(ct.ctx, ct.t.L(), ct.workloadNode)
+		if err != nil {
+			ct.t.Errorf("error getting grafana node external ip: %s", err)
+		}
+		ct.t.Status(fmt.Sprintf("started grafana at http://%s:3000/d/928XNlN4k/basic?from=now-15m&to=now", nodeURLs[0]))
+	} else {
+		ct.t.Status("skipping grafana installation")
 	}
-	nodeURLs, err := ct.cluster.ExternalIP(ct.ctx, ct.t.L(), ct.workloadNode)
-	if err != nil {
-		ct.t.Errorf("error getting grafana node external ip: %s", err)
-	}
-	ct.t.Status(fmt.Sprintf("started grafana at http://%s:3000/d/928XNlN4k/basic?from=now-15m&to=now", nodeURLs[0]))
 }
 
 type latencyTargets struct {
@@ -905,7 +907,7 @@ func registerCDC(r registry.Registry) {
 
 			exportStatsFile := ct.startStatsCollection()
 			feed := ct.newChangefeed(feedArgs{
-				sinkType: kafkaSink,
+				sinkType: cloudStorageSink,
 				targets:  allTpccTargets,
 				opts:     map[string]string{"initial_scan": "'only'"},
 			})

--- a/pkg/cmd/roachtest/tests/latency_verifier.go
+++ b/pkg/cmd/roachtest/tests/latency_verifier.go
@@ -90,7 +90,7 @@ func (lv *latencyVerifier) noteHighwater(highwaterTime time.Time) {
 	}
 
 	latency := timeutil.Since(highwaterTime)
-	if latency < lv.targetSteadyLatency/2 {
+	if lv.targetSteadyLatency == 0 || latency < lv.targetSteadyLatency/2 {
 		lv.latencyBecameSteady = true
 	}
 	if !lv.latencyBecameSteady {

